src/components/modal/walletModal.tsx

```diff
@@ -0,0 +1,156 @@
+import styled, { css } from 'styled-components';
+import { Modal } from './modal';
+import { fontSize, mediaQuery, pxToRem, spacing } from '../../styles';
+import { Button } from '../button/button';
+import { JazzIcon } from '../wallet/wallet';
+import { useState } from 'react';
+import { ButtonEnum, buttonType, useStore } from '../../helpers';
+import { useEthers } from '@usedapp/core';
+
+type Props = {
+	showModal: boolean;
+	setShowModal: (showModal: boolean) => void;
+	account: string;
+	isCopied?: boolean;
+};
+
+const ModalWrapper = styled.div`
+	display: flex;
+	align-items: center;
+	padding: ${spacing[18]} ${spacing[46]} 0 ${spacing[26]};
+
+	${mediaQuery('xs')} {
+		flex-direction: column;
+		align-items: flex-start;
+		padding: ${spacing[18]} ${spacing[46]} 0 ${spacing[14]};
+		margin-bottom: ${spacing[8]};
+	}
+`;
+
+const ModalContainer = styled.div`
+	margin-right: ${spacing[60]};
+`;
+
+const AccountTitle = styled.div
+(() => {
+	const { state: { theme } } = useStore();
+
+	return css`
+		font-size: ${fontSize[16]};
+		color: ${theme.font.pure};
+		line-height: ${spacing[22]};
+		margin-bottom: ${spacing[28]};
+	`;
+});
+
+const StatusContainer = styled.div`
+	margin-bottom: ${spacing[6]};
+`;
```

Comment:
very smart solution to only add `margin-bottom`to the `StatusContainer` and the `AccountTittle`